src/renderers/testing/ReactTestRendererFiberEntry.js

```diff
@@ -249,6 +249,11 @@ var TestRenderer = ReactFiberReconciler({
   useSyncScheduling: true,
 
   getPublicInstance,
+
+  now(): number {
+    // Test renderer does not use expiration
```

Comment:
@sebmarkbage I went back and forth on whether this should be an optional method. Landed on making it required because all renderers will need to account for features like expiration boundaries.